### PR TITLE
Add voice broadcast seek 30s forward/backward buttons

### PR DESCRIPTION
**Pull request by weeman1337**
Part of https://github.com/vector-im/element-web/issues/23282

PSF-1587

image

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: test/voice-broadcast/components/molecules/__snapshots__/VoiceBroadcastPlaybackBody-test.tsx.snap [hunk 1]
```diff
@@ -326,7 +485,7 @@ exports[`VoiceBroadcastPlaybackBody when rendering a stopped broadcast and the l
       <span
         class="mx_Clock"
       >
-        00:42
+        23:42
```
Comment (turt2live) [inline test/voice-broadcast/components/molecules/__snapshots__/VoiceBroadcastPlaybackBody-test.tsx.snap:488]: This implies an unexpected change?

**Review by turt2live — COMMENTED** [review]
I'm a bit surprised to see voice broadcast in its own tree as well - we likely aren't applying the correct CI checks to the tree. It should be merged into the existing `src/components` structure (in another PR).

File: test/voice-broadcast/components/molecules/__snapshots__/VoiceBroadcastPlaybackBody-test.tsx.snap (hunk 1 above)
Comment (weeman1337) [reply test/voice-broadcast/components/molecules/__snapshots__/VoiceBroadcastPlaybackBody-test.tsx.snap:488]: No, this is expected. I've added another test to explicitly look for the time update.

https://github.com/matrix-org/matrix-react-sdk/pull/9592/files#diff-6e97d35a4cfa5d17d4be2ea0e889b83b6081db3fbd23c7cc07333324837b14e8R135

comes from here

https://github.com/matrix-org/matrix-react-sdk/pull/9592/files#diff-6e97d35a4cfa5d17d4be2ea0e889b83b6081db3fbd23c7cc07333324837b14e8L107

**Comment by weeman1337** [comment]
> I'm a bit surprised to see voice broadcast in its own tree as well

As far as I remember we wanted to keep voice broadcast separated from the rest for now.

> we likely aren't applying the correct CI checks to the tree

Are there any checks not running or running differently for that part?

**Comment by turt2live** [comment]
> Are there any checks not running or running differently for that part?

I believe there's some checks which only run in `src/components`, but I may be mistaken (or they were removed). Either way, would be good to put the code into the rest of the tree, even if it is "temporary".

**Review by turt2live — APPROVED** [review]

**Pull request merged**